auth/README.md

```diff
@@ -352,7 +370,7 @@ represented in the following diagram:
 
 ![FirebaseUI authentication flow on Android](flow.png)
 
-### UI customization
+## UI customization
```

Comment:
This header is part of the `Table of Contents` so it should be on the same level as `configuration` and `using-firebaseui-for-authentication`.